using_redcap_api/using_redcap_api.md

```diff
@@ -0,0 +1,436 @@
+
+<div class = "important">
+
+If you've never used REDCap, there are [good instructional videos](https://projectredcap.org/resources/videos/) offered by Vanderbilt to help you understand and use REDCap.  One video that's particularly useful is [a fifteen minute video](https://redcap.vanderbilt.edu/consortium/videoplayer.php?video=redcap_overview03.mp4) that gives a high level overview of what REDCap is and how to use it.  That video is accompanied by a [.pdf transcript](https://redcap.vanderbilt.edu/consortium/redcap_video_transcripts/redcap_overview03.pdf).  If you're brand new to REDCap, please watch this video before proceeding.
+
+If you already have experience with REDCap, feel free to skip watching the video, as it goes over things you probably already know!
+
+</div>
+
+### Learn More About REDCap at your Institution
+
+Vanderbilt University provides REDCap software free of charge to institutions that join the REDCap Consortium.  It's currently in use by over 5,000 institutions and is used to host data for well over a million separate projects with millions of human users.  REDCap is offered to institutions, not to individual scientists or consumers, and participating institutions are responsible for their own REDCap software updates, training, and access for their users.  
+
+Research centers, clinics, and colleges and universities will have a dedicated server that they install REDCap on, and they'll make the REDCap website on their server available to their users (students, scientists, etc.).  Each institution does REDCap a little differently.  
+
+If REDCap is offered by your organization (your university or workplace), questions about how to use it effectively or solve problems like the site being down should go to the people at your institution responsible for REDCap.  Sometimes, an institution will have a big staff dedicated to REDCap, people who can offer all kinds of services like helping you design instruments and manage your data with a lot of support.  Other institutions have only one administrator who keeps the REDCap site running and up to date but isn't responsible for helping users figure out how to use REDCap.  Most institutions are somewhere in the middle.
+
+Not sure whether your institution has REDCap?  You may find your institution in the [list of partners](https://projectredcap.org/partners/).  For example, here are a list of partners that start with the letter "A".  Each partner then has contacts listed with their email addresses (simply click "View Collaborators"), so you can reach out and ask to get more information.
```

Comment:
This is probably irrelevant but it isn't just "A" entries. 

I get why "5th Regional..." is ahead of the A's but UAB Huntsville at the top is really throwing me.